test/e2e/operator_test.go

```diff
@@ -284,3 +292,165 @@ u3YLAbyW/lHhOCiZu2iAI8AbmXem9lW6Tr7p/97s0w==
 
 	return cl.Create(context.TODO(), secret), secret
 }
+
+func TestClusterIngressScale(t *testing.T) {
+	cl, ns, err := getClient()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	name := "default"
```

Comment:
It just occurred to me... this is an e2e test running against an existing cluster which is the target of other tests (some of which may even be running in parallel). And the test is mutating the default ClusterIngress. Isn't that dangerous? If we need to test scaling through e2e, I wonder if we'd better create new ClusterIngresses to work against during tests like this?

Reply:
This is a test in the cluster-ingress-operator repository, so it runs for the e2e-aws-operator job; doesn't that run in a separate cluster from the e2e-aws job? Also, this test is scaling the clusteringress up by 1 and then down by 1 to the original value, never below the original value, so it shouldn't break anything (or if it does, that might indicate a separate issue).